gramex/handlers/mlhandler.py

```diff
@@ -262,13 +264,11 @@ def setup(cls, data=None, model={}, backend='sklearn', config_dir='', **kwargs):
         # elif backend == 'transformers':
         #     NLPHandler.fit(**kwargs)
         if backend != 'sklearn':
+            cls.model_backend = backend
```

Comment:
Do we need this extra variable? `backend` can imply be assigned to `cls.backend`.